pom.xml

```diff
@@ -159,7 +159,7 @@
                   <message>The property `SSO_AUTH_SERVER_URL` is missing!</message>
                 </requireProperty>
               </rules>
-              <fail>true</fail>
+              <fail>false</fail>
```

Comment:
Why not simply removing the plugin from here?

Reply:
Since the property is required for the deployment to work, I thought keeping it here would reinforce that statement.

Reply:
ok ok!